static/app/types/react-router.d.ts

```diff
@@ -16,6 +16,13 @@ declare module 'react-router' {
     routes: PlainRoute[];
   }
 
+  interface RouteContextInterface {
```

Comment:
```suggestion
  interface RouteContextInterface {
```

No need to name interface, we dont care about conventions like IRouterContext or RouterContextInterface etc.

Reply:
Actually, can we not redeclare a new type here and maybe just pull whatever we can get from react-router types? 
I think `WithRouterProps` from static/app/types/react-router.d.ts is what you want

Reply:
@JonasBa react-router just types the arg in `render` method as `any`